README.md

```diff
@@ -96,6 +96,7 @@ More information on contributing and the general code of conduct for discussion
 | Playlist Exchange                    | [Playlist Exchange](https://github.com/DhanushNehru/Python-Scripts/tree/master/Playlist%20Exchange)                                              | A Python script to exchange songs and playlists between Spotify and Python.   
 | Pigeonhole Sort                      | [Algorithm](https://github.com/DhanushNehru/Python-Scripts/tree/master/PigeonHole)                                                            | The pigeonhole sort algorithm to sort your arrays efficiently!
 | PNG TO JPG CONVERTOR                    | [PNG-To-JPG](https://github.com/DhanushNehru/Python-Scripts/tree/master/PNG%20To%20JPG)                                              | A PNG TO JPG IMAGE CONVERTOR. 
+| Pomodoro Timer                    | [Pomodoro Timer](https://github.com/DhanushNehru/Python-Scripts/tree/master/Pomodoro Timer)                                              | A Pomodoro timer 
```

Comment:
Could you please fix the link and the label?? 
![Screenshot_20241006-110232_Chrome](https://github.com/user-attachments/assets/8239a62a-2801-4dc7-8dcb-34de8bf061ed)

Reply:
I hadn't noticed this! Thanks for pointing it out! It should be fixed now :)